trl/trainer/online_dpo_trainer.py

```diff
@@ -469,22 +489,25 @@ def training_step(self, model: nn.Module, inputs: Dict[str, Union[torch.Tensor,
         loss = losses.mean()
 
         # Log everything
+        if self.judge is None:
+            scores_margin = scores[chosen_indices] - scores[rejected_indices]
+            self.stats["objective/scores_margin"].append(self.accelerator.gather(scores_margin.mean()).mean().item())
+            self.stats["objective/scores"].append(self.accelerator.gather(scores.mean()).mean().item())
         self.stats["val/contain_eos_token"].append(contain_eos_token.float().mean().item())
         self.stats["logps/chosen"].append(self.accelerator.gather(chosen_logprobs_sum).mean().item())
         self.stats["logps/rejected"].append(self.accelerator.gather(rejected_logprobs_sum).mean().item())
-        self.stats["objective/scores"].append(self.accelerator.gather(scores.mean()).mean().item())
+
         kl = logprobs - ref_logprobs
         mean_kl = kl.sum(1).mean()
         self.stats["objective/kl"].append(self.accelerator.gather(mean_kl).mean().item())
         non_score_reward = (-self.beta * kl).sum(1)
         mean_non_score_reward = non_score_reward.mean()
         self.stats["objective/non_score_reward"].append(self.accelerator.gather(mean_non_score_reward).mean().item())
-        rlhf_reward = scores + non_score_reward
-        self.stats["objective/rlhf_reward"].append(self.accelerator.gather(rlhf_reward).mean().item())
+        if self.judge is None:
```

Comment:
Equivalent, but clearer this way imo
```suggestion
        if self.reward_model is not None:
```